ambry-api/src/main/java/com/github/ambry/quota/QuotaUsageLevel.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2021 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+/**
+ * Quota warning levels.
+ * The limit for each type of warning should be determined by {@link ThrottlePolicy} implementation.
```

Comment:
`The limit for each level should be determined by ...`

Reply:
done.